scripts/code_format.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
I think you want a `-e` on this so that any failing command sends its exit code back to the shell that ran the script.